src/main/java/no/difi/sdp/client2/internal/EbmsForsendelseContainer.java

```diff
@@ -0,0 +1,21 @@
+package no.difi.sdp.client2.internal;
+
+import no.digipost.api.representations.EbmsForsendelse;
+
+public class EbmsForsendelseContainer {
```

Comment:
Kan vi finne et annet navn på denne også? Dette er også et slags aggregatobjekt som bundler en EBMS-forsendelse og en dokumentpakke. Da representerer det egentlig ikke noe fra EBMS-domenet lenger, men en egen abstraksjon hvis en bestanddel er noe EBMS-relatert (en `EbmsForsendelse`). Kan vi se litt på hvordan denne brukes for å komme på et nytt navn?